### DISCUSSION TITLE
API's directly to excel

### DISCUSSION THREAD

**Discussion by Jurgens92** [kimai/kimai#4209] [General]
Hi,

is there a way I can pull data from Kimai directly to excel?

eg. pull data for "customer" or "user"

thank you

**Comment by treverehrfurth** [comment]
I'm also wondering if I can do an API call to directly get an excel/csv file returned!
@kevinpapst  is this possible or do we have to build our own excel export from the json?

**Reply by kevinpapst** [reply]
It is a JSON API. Often with multiple levels, so you need to convert it yourself into the required structure.

**Reply by treverehrfurth** [reply]
Thanks for the confirmation Kevin!
